File: Empeño/WindowsApplication1/conexion.vb
Imports MySql.Data.MySqlClient
Imports System.Data

Public Class conexion
    Public cn As MySqlConnection = New MySqlConnection("Server=192.168.56.101; Database=Empeno; Uid=usuario; Pwd=123;")
    Public dr As MySqlDataReader

    Public Sub abrir()
        cn.Open()
    End Sub

    Public Sub cerrar()
        cn.Close()
    End Sub

    Public Sub consulta(ByVal query As String)
        Dim cmd As MySqlCommand = New MySqlCommand(query, cn)
        dr = cmd.ExecuteReader()
    End Sub

    Public Sub movimientos(ByVal query As String)
        Dim cmd As MySqlCommand = New MySqlCommand(query, cn)
        cmd.ExecuteNonQuery()
    End Sub
End Class

File: PrestAgendaSolucion/PrestAgenda/conexion.cs
using System;
using System.Globalization;
using MySql.Data.MySqlClient;

namespace PrestAgenda
{
    class conexion
    {
        public MySqlConnection cn = new MySqlConnection("Server=192.168.1.70; Database=prestamos; Uid=tonny; Pwd=12345;");

        public MySqlDataReader dr;

        public void abrir()
        {
            cn.Open();
        }

        public void cerrar()
        {
            cn.Close();
        }

        public void movimiento(string query)
        {
            MySqlCommand cmd = new MySqlCommand(query, cn);
            cmd.ExecuteNonQuery();
        }

        public void consulta(string query)
        {
            MySqlCommand cmd = new MySqlCommand(query, cn);
            dr = cmd.ExecuteReader();
        }
    }
}
